sale_order_type_engine_data/models/sale_order.py

```diff
@@ -0,0 +1,12 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    engine_ref = fields.Char(string="Engine number")
+    engine_hours = fields.Integer(string="Engine hours")
```

Comment:
These fields are already defined in base addon `sale_order_engine_data`, so we shouldn't redefine again, unless they have to be changed (and it's not the case). So, remove them.